archCompare/compare_command.py

```diff
@@ -37,12 +43,11 @@ def main():
                            json file's diffs section \
                            Note- command line option if set overrides default reportsOn \
                            values in json config file ]")
-
     optional.add_argument("-r", "--remove_tmp", type=str, dest="remove_tmp", required=False,
                           default='y', help="remove tmporary data, default is 'y'")
 
     optional.add_argument("-v", "--version", action='version', version='%(prog)s ' + version)
-    optional.add_argument("-q", "--quiet", action="store_false", dest="verbose", default=True)
+    optional.add_argument("-q", "--quiet", action="store_false", dest="quiet", default=True)
```

Comment:
This stored param could be `verbose` rather than `quiet`, then negation later isn't needed.  Leaving the command line arg as `--quiet` though.

Reply:
done